server/cmd/mmctl/commands/user.go

```diff
@@ -251,6 +252,45 @@ var MigrateAuthCmd = &cobra.Command{
 	RunE: withClient(migrateAuthCmdF),
 }
 
+var PreferenceCmd = &cobra.Command{
+	Use:     "preference",
+	Aliases: []string{"pref"},
+	Short:   "Manage user preferences",
+}
+
+var PreferenceListCmd = &cobra.Command{
+	Use:     "list [--category category] [users]",
```

Comment:
I started to add comments to standardize the format of the arguments (brackets for optional, `<>` for positional, etc), but I realise that we're not applying one across the rest of the tool. I'll create a ticket to tackle this outside this PR